collections/Laud_Misc/MS_Laud_Misc_133.xml

```diff
@@ -38,8 +37,33 @@
                   </altIdentifier>
                </msIdentifier>
                <physDesc>
-                  <p n="composite">Composite: A || B || C</p>
+                  <objectDesc form="codex">
+                     <supportDesc material="perg">
+                        <support>parchment</support>
+                        <extent>i (early modern parchment) + 157 + i (early modern parchment) + i (early modern paper) leaves <dimensions type="leaf" unit="mm">
+                              <height>315</height>
+                              <width>245</width>
+                           </dimensions>
+                        </extent>
+                        <foliation>Modern foliation.</foliation>
+                        <collation>1–3<hi rend="superscript">8</hi> (fols. 1–24), 4<hi rend="superscript">10</hi> (fols. 25–34), 5–10<hi rend="superscript">8</hi> (fols. 35–82), 11<hi rend="superscript">8–1</hi> (8th cancelled; fols. 83–89) | 12–17<hi rend="superscript">8</hi> (fols. 90–137), 18<hi rend="superscript">6</hi> (fols. 138–143) | 19<hi rend="superscript">4+1</hi> (1st added; fols. 144–148), 20<hi rend="superscript">10–1</hi> (1st cancelled). <signatures>Quires 1–3 signed ‘Q ·i·’–‘·iii·’; 5–9 signed v–viiij; 16–17 signed v–vi.</signatures></collation>
+                     </supportDesc>
+                  </objectDesc>
+                  <handDesc>
+                     <handNote script="minusculeCaroline">Carolingian minuscule, described in detail by Bischoff, <title>Lorsch im Spiegel seiner Handschriften</title> (Munich, 1974), pp. 110–113.</handNote>
+                  </handDesc>
+                  <bindingDesc>
+                     <binding when="1638">
+                        <p>Standard Laudian binding; brown tanned calf over laminated pulpboard. Cloth ties excised.</p>
+                     </binding>
+                  </bindingDesc>
                </physDesc>
+               <history>
+                  <origin><orgName key="org_143763943">Lorsch Abbey</orgName>; written there according to Bischoff (<q>Jüngerer Lorscher Stil</q>). Certainly there during the Cistercian interlude, 1232/34-ca. 1245: Becker, <title>Catalogi bibliothecarum antiqui</title> (Bonn, 1885), no. 37, pp. 182, 122–131.</origin>
```

Comment:
Not sure what the Becker reference relates to: does that evidence the 13th century situation does he only print the Carolingian Lorsch catalogues? There is a more recent edition of the catalogues by Haase (?) that should perhaps be cited instead.

Reply:
I think it would be good to comment if possible as well (either here and/or in collation) about the relationship between the three separate parts of this manuscript - were they together at Lorsch (presumably not?) or were they assembled at Eberbach?